### DISCUSSION TITLE
Set status bar text color on iOS in code

### DISCUSSION THREAD

**Discussion by JohnHDev** [dotnet/maui#7709] [Q&A]
We have a XF app that has the ability to change the status bar text color via DI with the following:
    UIApplication.SharedApplication.StatusBarStyle = UIStatusBarStyle.LightContent;

This is obsolete as of iOS 9 so we don't want to keep that going forward. My understanding is the 'right' way now is to override PreferredStatusBarStyle in the UIViewController and then call SetNeedsStatusBarAppearanceUpdate() on the view controller (I haven't tested this).

I haven't seen anything built into Maui to support this directly, but Im unclear how I can override this in the view controller than Maui is using.

Any advice?

Thanks
John

**Comment by Denny966** [comment]
I also need help please :)
